all shape and some motion measures

CassiaCai opened the pull request.

CassiaCai committed:
all shape and some motion measures

valentina-s commented:
You do not need the `measures_func` folder. Unless we decide to split the plotting.py from the modules with the measures. I see there is not so many functions in plotting so splitting does not seem that necessary.

CassiaCai committed:
adding measures functions

CassiaCai commented:
I added the measures functions (still need to fix docstrings etc). Removed the measures_func folder so we only have measures. Also added a utils folder (for accessing LENS2 data, maybe not necessary? but need this for the example notebook).

valentina-s commented:
Update actions versions in https://github.com/ocetrac/ocetrac/blob/main/.github/workflows/test.yaml

checkout@v4
cache@v4
setup-miniconda@v3
codecov@v5

CassiaCai committed:
updating test.yaml with versions

CassiaCai committed:
changing measures folder, release.yaml

valentina-s commented:
@CassiaCai utils folder should go under ocetrac. I.e. all code goes under ocetrac.

CassiaCai committed:
changing location of utils
